addons/optionsmenu/init_loadMainMenuBox.sqf

```diff
@@ -15,17 +15,18 @@
  */
 #include "script_component.hpp"
 
-if (missionNamespace getVariable [QGVAR(newsLoaded), false]) exitWith {};
-GVAR(newsLoaded) = true;
-
 //Need to load from profileNamespace because ace_settings might not be init if world = empty
 if (profileNamespace getVariable [QGVAR(showNewsOnMainMenu), true]) then {
-    ((_this select 0) displayCtrl 80086) ctrlShow true;
+    ((_this select 0) displayCtrl 80090) ctrlShow true;
 
-    private ["_version"];
-    _version = getText (configFile >> "CfgPatches" >> "ace_main" >> "versionStr");
-    ((_this select 0) displayCtrl 68) ctrlSetText format ["Installed Version: %1", _version];
-    ((_this select 0) displayCtrl 69) htmlLoad "http://ace3mod.com/version.html";
+    private _ace3VersionStr = (getText (configFile >> "CfgPatches" >> "ace_main" >> "versionStr")) select [0,5];
+    ((_this select 0) displayCtrl 80094) ctrlSetText format ["Version: %1", _ace3VersionStr];
+    ((_this select 0) displayCtrl 80095) htmlLoad "http://ace3mod.com/version.html";
```

Comment:
ACE3 itself is not compatible with older versions, and this is not exclusive to Apex owners.